### DISCUSSION TITLE
Sync problem

### DISCUSSION THREAD

**Discussion by krizsy** [Chia-Network/chia-blockchain#6226] [Support]
My node try to sync for a week now but its unable to comlete. It is always after about 20-30 minutes and status is not synced.
Port 8444 is open and I've added introducer-eu.chia.net also.
In the past I had no problem with sync.

**Comment by tiny0little** [comment]
please test tcp8444 on [portchecker.co](https://portchecker.co) and let us know the result
also try to add all introducers and node.chia.net

```
North Asia introducer-apne.chia.net:8444
South Asia introducer-apse.chia.net:8444
Western North America: introducer-or.chia.net:8444
Eastern North America introducer-va.chia.net:8444
Europe: introducer-eu.chia.net:8444
```

make sure they are added `chia show -c | egrep "FULL_NODE|INTRODUCER"`

**Reply by krizsy** [reply]
portchecker.co says port is open
![chia3](https://user-images.githubusercontent.com/85048991/120097755-978c1e00-c132-11eb-998c-0e0ddfbe4b75.jpg)
![chia2](https://user-images.githubusercontent.com/85048991/120097756-9824b480-c132-11eb-94f1-2516aca07e13.jpg)
![8444](https://user-images.githubusercontent.com/85048991/120097757-98bd4b00-c132-11eb-891c-f2f6229089d5.jpg)
![chia](https://user-images.githubusercontent.com/85048991/120097758-98bd4b00-c132-11eb-9bc7-2a67e1467c4b.jpg)

**Reply by tiny0little** [reply]
i can't see node.chia.net. try adding it again and again, until it will be in the list

**Reply by krizsy** [reply]
I could add it but nothing changed. After a few hours it disappeared from the list

**Reply by tiny0little** [reply]
does your peak height increase? does status change to syncing?
what version you run?

**Reply by krizsy** [reply]
peak height increases all the time but its always 30-60 minutes away from actual. Status is not synced all the time. Version is 1.1.4

**Reply by tiny0little** [reply]
try to upgrade to 1.1.6 first. most of hosts are on latest version, they simply to do want to talk to your node due to version mismatch

**Reply by krizsy** [reply]
It worked. Thank you for your help!

**Comment by BrandtH22** [comment]
@krizsy , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
